app/code/Magento/QuoteGraphQl/Model/GetMaskedQuoteIdByReversedQuoteId.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\QuoteGraphQl\Model;
+
+use Magento\Quote\Model\ResourceModel\Quote as QuoteResource;
+use Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface;
+use Magento\Quote\Model\QuoteFactory;
+
+class GetMaskedQuoteIdByReversedQuoteId
```

Comment:
Please, reuse this class in all of the places (find by `getMaskedQuoteIdByReversedQuoteId`)
But not just in one testcase class

Also, this class is used just for testing, so need to move to test directory `dev/tests/api-functional/testsuite/Magento/GraphQl/Quote`
``

Reply:
@naydav , I am sorry but I have a question about **GetMaskedQuoteIdByReversedQuoteId** class name. 

We load quote by `reserved_order_id` field
```
$this->quoteResource->load($quote, $reversedQuoteId, 'reserved_order_id');
```
and in the same time new class is called as `GetMaskedQuoteIdByReversedQuoteId`. 

May we should use a proper name for it - `GetMaskedQuoteIdByReservedOrderId`?

Thank you!

CC: @AleksLi

Reply:
Hi @atwixfirster,
I saw the fix in your PR https://github.com/magento/graphql-ce/pull/450/files

But for avoiding a lot of merge conflicts I am going to merge all of related PRs and finally make the fix in your PR which will be merged last.

Reply:
I didn't cover everything in all the places, because I thought it might be something with the class name or so. I will do that now